web-portal/store/admin.js

```diff
@@ -41,7 +41,7 @@ export const actions = {
 
     ApiService.all([
       ApiService.get(CURRENT_EVENTS_NON_VERIFIED_PATH, idToken),
-      ApiService.get('/events/non-verified/tags/online-resource', idToken)
+      ApiService.get('/v1/events/non-verified?tags=online-resource', idToken)
```

Comment:
I didn't do it in this pass, but we should consider building the version segment into the API service.

Reply:
I'm confused. I thought I did that: `@Controller(`${VERSION_1_URI}/events`)`

Reply:
yes good idea.That should be part of the base url, so we don't have to update it everywhere. Honestly I think we can just do this in the api as part of the base url we already have can't we?